server/filters/index.js

```diff
@@ -1,7 +1,9 @@
 import parseBody from './parse-body';
+import youtubeEmbedUrl from './youtube-embed-url';
```

Comment:
Dang it! I need to get compilation on the server as it would have picked up on this.